### PR TITLE
Adds main chart

### PR DESCRIPTION
**Pull request by noahamac**

### CODE REVIEW

File: src/FunnelChart/types.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  overflow: hidden;
+
+  :hover {
+    filter: brightness(80%);
+  }
+`
+export const FunnelStepContents = styled.span`
+  display: table-cell; 
+  vertical-align: middle;
+  // TODO: Label Scale option
```
Comment (noahamac) [inline src/FunnelChart/types.ts:31]: TODOs here indicate where some config options should be hooked up

File: src/SteppedFunnelContainer.tsx [hunk 2]
```diff
@@ -65,16 +39,47 @@ const vis: SteppedFunnelChart = {
...
-    this.options.right_axis_label = config.label_right_axis && Vizzy.stringOption("Axes", "Right Axis Label", "", 0, "label_right_axis", vis)
+    this.options.left_axis_label = config.label_left_axis && Vizzy.dependString("Axes", "Left Axis Label", "", "label_left_axis", vis)
+    this.options.right_axis_label = config.label_right_axis && Vizzy.dependString("Axes", "Right Axis Label", "", "label_right_axis", vis)
 
     // register new options if options has changed since last render
     if (JSON.stringify(previousOptions) !== JSON.stringify(this.options)) {
       this.trigger && this.trigger('registerOptions', this.options)
     }
 
+    let inputRow = data[0]
```
Comment (noahamac) [inline src/SteppedFunnelContainer.tsx:50]: @TJ this is my very bare bones data/qr plumbing implementation :D

File: src/SteppedFunnelContainer.tsx [hunk 3]
```diff
@@ -65,16 +39,47 @@ const vis: SteppedFunnelChart = {
...
+    let chunkedData: Chunk[] = inputFields.map((fieldName: string) => {
+      let datum = inputRow[fieldName]
+      return {
+        label: fieldName,
+        value: datum.value,
+        rendered: datum.rendered,
+        links: datum.links as Link[]
+      }
+    })
+    config.autosort && chunkedData.sort((a: Chunk, b: Chunk) => {
```
Comment (noahamac) [inline src/SteppedFunnelContainer.tsx:62]: I found that moving fields in the data tab didn't change order of fields in viz until after a reload. is this true?

**Review by thomasbanghart — APPROVED** [review]
This is awesome!

**Pull request merged**